util/chplenv/compiler_utils.py

```diff
@@ -14,7 +14,9 @@
 @memoize
 def get_compiler_version(compiler):
     version_string = '0'
-    if 'gnu' in compiler:
+    if compiler == 'aarch64-gnu':
+        version_string = run_command(['aarch64-unknown-linux-gnu-gcc', '-dumpversion'])
```

Comment:
Is this actually the compiler name? To me it looks like it's just  `gcc-cross-aarch64`, also are we actually setting the target compiler to this, or are we still using the `cc` wrapper?